pkg/virt-handler/isolation/process_test.go

```diff
@@ -107,3 +108,44 @@ func (p ProcessStub) PPid() int {
 func (p ProcessStub) Executable() string {
 	return p.binary
 }
+
+// This test has been thankfully taken from the repo https://github.com/mitchellh/go-ps/
+func TestFindProcess(t *testing.T) {
```

Comment:
Can you "migrate" these tests into the Ginkgo testsuite? Do they even run currently?

Reply:
@0xFelix I haven't used `DescribeTable`, but have migrated them to Ginkgo.